rust/agama-lib/share/profile.schema.json

```diff
@@ -967,6 +929,30 @@
       "items": {
         "type": "object"
       }
+    },
+    "scripts": {
+      "title": "User-defined installation scripts",
+      "description": "User-defined scripts to run at different points of the installation",
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "pre": {
+          "title": "Pre-installation scripts",
+          "description": "User-defined scripts to run before the installation starts",
```

Comment:
Um, when exactly? Nothing done yet?

I mean later we might add a ` post-partitioning` script so it should be clean whether at this point something is already done by Agama or the system has not been touched yet.

Reply:
Before the post-partitioning. I can improve the description.

Reply:
They were expected to be used to modify the profile, but I am not sure whether we should implement that behavior. Or perhaps to activate some hardware.